src/ImageSharp/Formats/Webp/BitReader/Vp8BitReader.cs

```diff
@@ -186,7 +186,7 @@ private void LoadNewBytes()
     {
         if (this.pos < this.bufferMax)
         {
-            ulong inBits = BinaryPrimitives.ReadUInt64LittleEndian(this.Data.Memory.Span.Slice((int)this.pos, 8));
+            ulong inBits = BinaryPrimitives.ReadUInt64LittleEndian(this.Data!.Memory.Span.Slice((int)this.pos, 8));
```

Comment:
There's some horrible temporal coupling here. I don't think the correct fix would be to add `!`. It may require much deeper refactoring.

Reply:
Yeah probably. And there are some more classes where it is not so easy to remove the #nullable disable

Reply:
I'm gonna be looking at some optimization opportunities in those classes. I think we should leave this until that is done. 

We've covered all the public bits now anyway I think?

Reply:
ok I tried to ref it. Can you have a look?

Reply:
<img width="306" alt="image" src="https://user-images.githubusercontent.com/873751/220645638-573e52ad-1a3c-42b7-b574-9bfd5fe9d89e.png">
Those are the areas where nullable disable is still set

Reply:
> ok I tried to ref it. Can you have a look?

Ah nice, will do!

Reply:
Refactor looks good. I just pushed some additional cleanup.

Reply:
Actions seems to be having a wobble. I'll merge in the morning.